Support select slots in form-ripe

NFSS10 opened the pull request:
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-registry-ui/pull/16 |
| Dependencies | -- |
| Decisions | Need to use the select slots to be able to customize the select values UI<br>Example of slots in use below, take notice in the two selects that have their entries with an avatar, name and email|
| Animated GIF | <br>![client-create](https://user-images.githubusercontent.com/22588915/92007231-93ff6e00-ed3d-11ea-8b9d-0219f01ddddf.gif) |

NFSS10 commented:
Need to check things

joamag commented:
@NFSS10 Please elaborate on the use-case of this change... use a screenshot if necessary

NFSS10 commented:
> @NFSS10 Please elaborate on the use-case of this change... use a screenshot if necessary

Updated decisions field and added an example of client create, where we have selects with custom entries (image, name and email)
